### ISSUE TITLE
No validation accuracy should be represented by None not nan

### ISSUE THREAD

**Issue by TobyRoseman** [apple/turicreate#1941]
If you train a drawing classifier without a validation set, the validation accuracy gets saved as a `nan`. I think `None` makes a lot more sense.
